eth/handler.go

```diff
@@ -606,13 +606,30 @@ func (h *handler) BroadcastBlock(block *types.Block, propagate bool) {
 			log.Error("Propagating dangling block", "number", block.Number(), "hash", hash)
 			return
 		}
+
+		// These are the static and trusted peers which are not
+		// in `transfer := peers[:int(math.Sqrt(float64(len(peers))))]`
+		staticAndTrustedPeers := []*ethPeer{}
+
+		for _, peer := range peers[int(math.Sqrt(float64(len(peers)))):] {
```

Comment:
Not sure about this, just asking: shouldn't this cycle go over the whole range of peers?

Reply:
So this loop is to only get those static and trusted peers which are not eligible for the legacy broadcast. The peers we are skipping here are already included below
```
transfer := peers[:int(math.Sqrt(float64(len(peers))))]
```

I have mentioned this in the comment
```
// These are the static and trusted peers which are not
// in `transfer := peers[:int(math.Sqrt(float64(len(peers))))]`
staticAndTrustedPeers := []*ethPeer{}
```